pkgs/tools/nix/fh/default.nix

```diff
@@ -28,12 +29,17 @@ rustPlatform.buildRustPackage rec {
 
   buildInputs = lib.optionals stdenv.isDarwin [
     darwin.apple_sdk.frameworks.Security
+    darwin.apple_sdk.frameworks.SystemConfiguration
     gcc.cc.lib
   ];
 
-  env = lib.optionalAttrs stdenv.isDarwin {
-    NIX_CFLAGS_COMPILE = "-I${lib.getDev libcxx}/include/c++/v1";
-  };
+  env =
+    {
+      SSL_CERT_FILE = "${cacert}/etc/ssl/certs/ca-bundle.crt";
```

Comment:
Is adding `cacert` to `buildInputs` not enough?

Reply:
Seems to be enough, fixed